boards/others/candlelightfd/candlelightfd_stm32g0b1xx_dual_defconfig

```diff
@@ -0,0 +1 @@
+CONFIG_GPIO=y
```

Comment:
Not blocking, but we usually recommend enabling MPU and HW stack protection when available:
```
# Enable MPU
CONFIG_ARM_MPU=y

# Enable HW stack protection
CONFIG_HW_STACK_PROTECTION=y
```